meson.build

```diff
@@ -18,9 +18,10 @@ i18n = import('i18n')
 
 
 # glib version info
-glib_version = '2.56'
+glib_version = '2.70'
 glib_max_version = glib_version
-glib_min_version = glib_version
+# While we actually depend on 2.70 because of libsoup3 we have some deprecated usage that we'll deal with later so this silences it for now.
```

Comment:
Shouldn't libsoup handle this check implicitly when it's built? There's nothing in trg that requires it though obviously in gtk/libsoup it may require a different API version.

Reply:
Yes, you could change it to 2.66 if you prefer. At the end of the day though TRG will always have glib 2.70+, so we can simply use all APIs up to that point.

Reply:
Ah, so this can avoid the deprecation warnings? Interesting. I'll add the GUri code then.

Reply:
Also as an aside: now that unix socket support is merged into transmission proper, GUri code in the future looks like it can greatly simplify handling a http unix socket connection.

Reply:
Also while 2.70 probably isn't our real dep, 2.56 isn't either. GUri is in headers we include.

Reply:
Just an FYI, libsoup does support HTTP over unix sockets: https://github.com/GNOME/libsoup/blob/689e072db546042cf0f0681f8a0cf653075d1860/tests/unix-socket-test.c#L41-L43

Reply:
Yeah I did see that, it was one of the deciding factors in switching to it. Curl can do it as well, but adding curl-multi to the mainloop would have been harder than this.